Alignment/OfflineValidation/test/inspectData_cfg.py

```diff
@@ -69,7 +69,7 @@
 from Configuration.PyReleaseValidation.upgradeWorkflowComponents import upgradeProperties
 ConditionsInfo = {}
 if 'D' in options.Detector:
-    ConditionsInfo = upgradeProperties[2026][options.Detector] # so if the default changes, change wf only here
+    ConditionsInfo = upgradeProperties['Run4'][options.Detector] # so if the default changes, change wf only here
```

Comment:
because of this , you need to change also the input string here:

https://github.com/cms-sw/cmssw/blob/7d3d80e8707194c87090fd2ef9ae0319ea2cefa1/Alignment/OfflineValidation/test/testingScripts/test_unitMiscellanea.sh#L8